### ISSUE TITLE
Unable to import workspace with deegree-webservices-3.3.13 war

### ISSUE THREAD

**Issue by awst-baum** [deegree/deegree3#510]
When installing the deegree-webservices-3.3.13 war file on my machine (tomcat 7.0.42, openSUSE 13.1), I have problems following the getting started doc at "Downloading and activating example workspaces" (http://download.deegree.org/documentation/3.3.13/html/lightly.html#downloading-and-activating-example-workspaces):
When I click on "deegree-workspace-wps", I get an error message: "Unable to import workspace: /usr/share/tomcat/.deegree/deegree-workspace-wps/processes/ContainsProcess.xml (No such file or directory) "
My working directory is /usr/share/tomcat/.deegree - which does not exist.
![Uploading degree3_workspaces.png . . .]()

So how do I rectify that?

**Comment by tfr42** [comment]
Have you checked if the user running the Apache Tomcat process has read and write access to the `/usr/share/tomcat/.deegree`directory?

**Comment by awst-baum** [comment]
I created the /usr/share/tomcat/.deegree directory and gave it to the tomcat user and now I can import workspaces - thanks :)
Is there a way to configure the webservice to put the deegree dirctory somewhere else?

**Comment by tfr42** [comment]
Yes, as described in chap.  [4.2. Location of the deegree workspace directory] [1] you can set the environment variable `DEEGREE_WORKSPACE_ROOT`and point this to an arbitrary directory (tomcat process needs read/write access). Hope that answers your question.


[1]: http://download.deegree.org/documentation/3.3.14/html/basics.html#location-of-the-deegree-workspace-directory

**Comment by awst-baum** [comment]
Yes, it does - thanks a lot for the quick help! :)

Maybe you could add a few sentences to the manual, Section 3.1.1. "Downloading and activating example workspaces", something like:
If you have trouble importing workspaces, check your working directory (first line of the workspace management screen). If it doesn't exist, create it (it will be filled automatically once you import workspaces); if it exists, check access permissions. If you want to change the location of the working directory, refer to Section 4.2. "Location of the deegree workspace directory" of the manual.
